optaplanner-examples/src/main/java/org/optaplanner/examples/flightcrewscheduling/persistence/FlightCrewSchedulingGenerator.java

```diff
@@ -190,16 +180,12 @@ private void createFlightList(FlightCrewSolution solution, int flightRoundTripsP
                         END_MINUTE_OF_DAY - flyingTime - START_MINUTE_OF_DAY + 1);
                 int arrivalMinute = departureMinute + flyingTime;
                 for (LocalDate date = firstDate; date.compareTo(lastDate) <= 0; date = date.plusDays(1)) {
-                    Flight flight = new Flight();
-                    flight.setId(flightId);
-                    flightId++;
-                    flight.setFlightNumber(flightNumber);
-                    flight.setDepartureAirport(departureAirport);
-                    flight.setDepartureUTCDateTime(date.atTime(departureMinute / 60, departureMinute % 60));
-                    flight.setArrivalAirport(arrivalAirport);
-                    flight.setArrivalUTCDateTime(date.atTime(arrivalMinute / 60, arrivalMinute % 60));
+                    Flight flight = new Flight(flightId, flightNumber, departureAirport,
+                            date.atTime(departureMinute / 60, departureMinute % 60), arrivalAirport,
+                            date.atTime(arrivalMinute / 60, arrivalMinute % 60));
```

Comment:
Local variables with the results of these calculations would improve readability.